ebuild-writing/common-mistakes/text.xml

```diff
@@ -504,8 +504,29 @@ the ebuild's quality and ease review.
 </p>
 </body>
 </subsection>
-</section>
 
+<subsection>
```

Comment:
How does it differ from any other prefixed tools (e.g. CC/NM/STRING/so on) and why should we use a separate section for this?

Reply:
Well, we can change it to "calling prefixed tools directly". But this is a particularly annoying issue which has far less attention (I've had to fix a lot of examples of it).